mantaray/config.py

```diff
@@ -217,6 +219,20 @@ def __init__(
             )
             self._rownumber += 1
 
+        self._away_notify_var = tkinter.BooleanVar(value=initial_config["away_notify"])
+        if connecting_to_new_server:
+            self.away_notify_checkbox = None
+        else:
+            self.away_notify_checkbox = ttk.Checkbutton(
+                self,
+                text="Notify me about other users' Away status",
```

Comment:
I think we can just always request the `"away-notify"` capability. If the server doesn't have it, we won't be able to update the user list colors. I don't see any reason why you would want to disable that, so we no longer need a checkbox in settings.